test/unit/test_taxon_filter.py

```diff
@@ -37,7 +37,7 @@ def setUp(self):
         TestCaseWithTmp.setUp(self)
         self.polio_fasta = os.path.join(
             util.file.get_test_input_path(),
-            'TestMetagenomicsViralMix', 'db', 'library', 'Viruses', 'Poliovirus_uid15288', 'NC_002058.ffn'
```

Comment:
I just noticed this, why did you change the input to a unit test of an unrelated piece of code?

Reply:
Oh I see, github diff doesn't show the relationship between the old .ffn and the new .fna.